assets/charts/control-plane/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -1,57 +1,23 @@
-# On platforms like AWS, it's enough for kube-apiserver to listen on Node InternalIP. On other
```

Comment:
There is more code which can be removed. More details here: https://gist.github.com/invidian/d77dca57bcf3dbe49fa9b18330b5366a.

Reply:
done.